uint8 RANGE_SENSOR_TYPE_ULTRASONIC = 1
uint8 RANGE_SENSOR_TYPE_INFRARED = 1
uint8 RANGE_SENSOR_TYPE_TOF = 2
uint8 RANGE_SENSOR_TYPE_BUMPER = 3

uint8 type
RangeData[] data